src/components/emptyCart.tsx

```diff
@@ -1,20 +0,0 @@
-import React from 'react';
-
-
-
-
-function emptyCart(){
```

Comment:
I just forget to remove it pleaze do it ,it doesn't have any utility right now .